pkg/cmd/controller/cmd.go

```diff
@@ -0,0 +1,19 @@
+package controller
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+
+	"github.com/openshift/service-ca-operator/pkg/controller"
+	"github.com/openshift/service-ca-operator/pkg/version"
+)
+
+func NewController() *cobra.Command {
+	cmd := controllercmd.
+		NewControllerCommandConfig("service-ca-controller", version.Get(), controller.StartServiceCAControllers).
+		NewCommand()
+	cmd.Use = "controller"
```

Comment:
Nit: `ca-injectors`? `controller` is just too vague

Reply:
It's not just ca injectors.  It's the cert secret generating controller too. The new normal for this operator is an 'operator' command and the 'controller' command that it manages.

Reply:
My bad, I've gone through the PR a bit too fast. Name's fine, then.